framework/src/engine/consensus/consensus.ts

```diff
@@ -366,13 +366,6 @@ export class Consensus {
 		this._commitPool.addCommit(singleCommit, true);
 	}
 
-	public async getMaxRemovalHeight(): Promise<number> {
```

Comment:
Is the same function still defined somewhere else? It would still be needed to decide when a single commit that is stored locally can be safely removed.

Reply:
yes, the function is defined for the certificate generation. This was a duplicate function